.travis.yml

```diff
@@ -11,27 +11,33 @@ php:
   - 5.6
   - 7.0
   - 7.1
+  - 7.2
   - nightly
 
 matrix:
   include:
     - php: 5.5
       env: deps=low SYMFONY_DEPRECATIONS_HELPER=weak
     - php: 5.6
-      env: SYMFONY_VERSION="2.8.*@dev"
-    - php: 5.6
+      env: SYMFONY_VERSION="2.8.*"
+    - php: 7.0
+      env: SYMFONY_VERSION="3.2.*"
+    - php: 7.1
+      env: SYMFONY_VERSION="3.4.*@beta"
+    - php: 7.2
+      env: SYMFONY_VERSION="4.0.*@beta"
```

Comment:
-1 for that. This is already covered by the `deps=dev` job. I'm only adding version-specific jobs for LTS versions (I don't want to update this config file every 6 months to change the versions, too painful)

Reply:
Which deps=dev? The one which was allowed to fail? We can't afford for this bundle to stay incompatible with next Symfony version and figure it out only at RC phase again. It must have been a while since tests started to fail for Symfony 4 and it was ignored. These constraints should be set in the way it's clear that something is wrong when tests fail, but on the other side aren't supposed to be expected to fail randomly, which would be the case with dev dependencies. I suggest to have matrix for latest dev=alpha then. That's when failures are starting to be unwanted.